packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -173,31 +220,32 @@ const renderColumn = (displayGroup, displayGroupIndex, emphasis, overrideDefault
   );
 };
 
-const renderView = (displays, layout, emphasis, overrideDefaultStyling) => {
+const renderView = (displays, layout, emphasis, overrideDefaultStyling, trueColumn) => {
   if (displays === null || displays === undefined || !displays.length) {
     return undefined;
   }
 
-  const primaryColumn = [];
-  const displayGroups = [];
+  let displayGroups = [];
   const displaysSlice = displays.slice(0, 8);
+  const primaryColumn = [];
 
   if (layout === Layouts.TWO_COLUMNS) {
-    let count = 0;
-    const secondaryColumn = [];
-
-    while (displaysSlice.length) {
-      count += 1;
-
-      if (count % 2 === 0) {
-        secondaryColumn.push(displaysSlice.splice(0, 1));
-      } else {
-        primaryColumn.push(displaysSlice.splice(0, 1));
+    if (trueColumn) {
+      displayGroups = twoColumnGrouping(displaysSlice);
+    } else {
+      while (displaysSlice.length) {
+        displayGroups.push(displaysSlice.splice(0, 2));
```

Comment:
Isn't this pushing the same array information to the array multiple times?

Reply:
No, splice removes the first two displays from displaysSlice and pushes them together to displayGroups. So each time it's taking the top two off and pushing them. It's the same logic that was being used previously.

So displayGroups would look like [[display 1, display 2], [display 3, display 4], [display 5, display 6], [display 7, display 8]]. And by the end displaysSlice would be empty

Reply:
Ah right, `spice` mutates the original array as opposed to `slice`. +1